### ISSUE TITLE
Permit PUT request error, blocking email

### ISSUE THREAD

**Issue by mtlaney** [18F/fs-intake-module#1549]
## Notes
* After completing pay.gov completeCollection on a new firewood permit, we're receiving a 400 error on the permit put request to store the pay.gov token. This is also blocking the execution of the email follow-up*

## Acceptance Criteria
- [x] Permits successfully PUT after pay.gov transaction
- [ ] Email client fires off permit details to customer

## Tasks
- [x] Correct Permit PUT either routes or parameters. 
- [x] Ensure Permit PUT is firing off after pay.gov transaction

## Definition of Done
- [x] Pull requests meet technical definition of done
- [x] Compare finished design with mockup
- [x] Usability tested

**Comment by mwreiss** [comment]
@mtlaney  can you verify this one is good to be completed/validated?

**Comment by mtlaney** [comment]
Waiting on the puppeteer install on staging

**Comment by mtlaney** [comment]
closing this one as the dev side engineering has been complete. PUT request is now complete. Will open another ticket referencing puppateer install
